hal/hal_i2c_host.h

```diff
@@ -87,8 +89,8 @@ typedef enum {
  *                                                                          - IRQ priority
  */
 uhal_status_t i2c_host_init(const i2c_periph_inst_t i2c_peripheral_num, const i2c_clock_sources_t clock_sources,
```

Comment:
It might be useful to take the autoformatter on this whole file (CTRL+ALT+L in CLion)